### ISSUE TITLE
Failed to import transformers.trainer because of the following error (look up to see its traceback): module 'numpy.linalg._umath_linalg' has no attribute '_ilp64'

### ISSUE THREAD

**Issue by hugoquintel** [georgian-io/Multimodal-Toolkit#71]
I think the latest version of numpy cause this bug since downgrading numpy to version 1.24.1 seem to fix the issue.

**Comment by akashsaravanan-georgian** [comment]
Hi @hugoquintel, thanks for flagging this. What version of numpy were you using prior to downgrading?

**Comment by hugoquintel** [comment]
The default version on google colab i believe

**Comment by akashsaravanan-georgian** [comment]
Thanks! I'll look into this and fix it for our next release.

**Comment by akashsaravanan-georgian** [comment]
Hi @hugoquintel it seems like this is an issue with Google CoLab. I was able to recreate the issue you faced. For some reason the version installed does not match the version that shows up when we run `np.__version__`. 

Upon further investigation, if you restart the kernel (Runtime -> Restart Session in the menu), this issue goes away. I'll add a line to the notebook mentioning this, but I don't think there's any other changes required to the library itself. 

Closing this issue but happy to re-open if the above steps don't solve it for you!
